modules/ROOT/pages/notifications/index.adoc

```diff
@@ -64,6 +64,11 @@ The following categories exist, ordered by severity:
 | Information provided while executing database and server related commands.
 |
 
+| SCHEMA
+| INFORMATION
+| Information provided while managing indexes and constraints.
```

Comment:
not sure if we want to say managing or what we want to say here?

Reply:
I think it's fine like that.